core/kernel.go

```diff
@@ -145,6 +146,30 @@ func NewKernel(ctx context.Context, gen *proposal.Genesis, conf *config.Config,
 				return serveProcess, nil
 			},
 		},
+
+		{
+			Name:    "GRPC",
+			Enabled: conf.GRPC.Enabled,
+			Launch: func() (process.Process, error) {
+				listen, err := net.Listen("tcp", conf.GRPC.ListenAddress)
+				fmt.Println("listen", listen)
+				if err != nil {
```

Comment:
If you need to log something use logger. Please try avoid fmt.Print(...) in final commits.